app/src/main/java/science/apolline/view/activity/MainActivity.kt

```diff
@@ -101,11 +103,40 @@ class MainActivity : RootActivity(), NavigationView.OnNavigationItemSelectedList
                 invalidateOptionsMenu()
             } else if (BluetoothLeService.ACTION_GATT_DISCONNECTED.equals(action)) {
                 mConnected = false
+
+                if(distroyed == false) {
+                    val builder = AlertDialog.Builder(this@MainActivity)
+
+                    builder.setTitle("capteur deconnecté")
+
+                    builder.setMessage("Le capteur a été deconnecté. Vérifiez qu'il est allumé et qu'il est assez proche de votre de téléphone.")
```

Comment:
Strings should be externalized.

Reply:
`Le capteur a été déconnecté. Vérifiez qu'il est allumé et qu'il est à proximité de votre téléphone.`